src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd

```diff
@@ -574,6 +575,18 @@
         </xsd:choice>
     </xsd:complexType>
 
+    <xsd:complexType name="http_client_retry_strategy">
+        <xsd:sequence>
+            <xsd:element name="http-codes" type="xsd:integer" minOccurs="0" maxOccurs="unbounded" />
```

Comment:
wrong name. That should be `http-code` as each element represents a single code (`->fixXmlConfig` is what ensures that one or multiple `http-code` elements always produce a single array `http_codes` with all the values in it)

Reply:
good catch, thank you